__tests__/plugin.test.ts

```diff
@@ -533,38 +533,16 @@ describe('babel plugin', () => {
       expect(code).toHaveWorkletData();
       expect(code).toMatchSnapshot();
     });
-  });
-
-  describe('for runOnUI', () => {
-    it('workletizes ArrowFunctionExpression inside runOnUI automatically', () => {
-      const input = html`<script>
-        runOnUI(() => {
-          console.log('Hello from the UI thread!');
-        })();
-      </script>`;
-
-      const { code } = runPlugin(input);
-      expect(code).toHaveWorkletData();
-      expect(code).toMatchSnapshot();
-    });
 
-    it('workletizes unnamed FunctionExpression inside runOnUI automatically', () => {
+    it('workletizes hook wrapped worklet reference automatically', () => {
```

Comment:
This should be in the referencing test suite

Reply:
Actually when I read it again it's in a good position 😆